src/Spectre.libClassifier.Tests/SvmTest.cpp

```diff
@@ -0,0 +1,71 @@
+/*
+* SvmTest.cpp
+* Tests Svm classifier
+*
+Copyright 2017 Spectre Team
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include <gtest/gtest.h>
+#include "Spectre.libClassifier/Types.h"
+#include "Spectre.libClassifier/OpenCvDataset.h"
+#include "Spectre.libClassifier/Svm.h"
+
+namespace
+{
+    using namespace Spectre::libClassifier;
+
```

Comment:
Number of support vectors has not been tested yet.

Reply:
done